docs/commit-pub-rand.md

```diff
@@ -0,0 +1,124 @@
+# Public Randomness Commit Specification
+
+## Overview
+
+The finality provider periodically commits public randomness to the consumer
+chain to be used for future block finalization. This document specifies the
+process of committing public randomness.
+
+## Commit Process
+
+### Generating a Commit
```

Comment:
I think I would describe here also:
- full commit message used in Babylon (can be link to Babylon proto file)
- the format of randomness i.e 32byte public point
- the merkle tree library used and format of the proof

In general, this spec should have enough details that any body can implement his own randomness committing process.

Reply:
Good point! Added useful links in [a73f745](https://github.com/babylonlabs-io/finality-provider/pull/117/commits/a73f7457cf1fd2f845935c7ed80b6351b2900209)